JS-week-2/candy_helper.js

```diff
@@ -0,0 +1,43 @@
+let boughtCandy = [];
+
+function addCandy(candyType, weight){
+    let cadies = ['sweet','chocolate','toffee','chewing_gum'];
+    let candyPricePerGram = [0.5, 0.7, 0.3, 0.03];
```

Comment:
An alternative way to store this data is to introduce an object like this:

```js
let candies = {
    'sweet': 0.5,
    'chocolate': 0.7,
    'toffee': 0.3,
    'chewing_gum': 0.03
};
```

This way, to get the price per gram of a specific candy you can write `candies['sweet']`. 

P.S. There is nothing wrong with your solution, and I think you should keep it as is because you already have the code for locating the element in these arrays written. Just wanted to bring up another method of storing data for this exercise 😉